pkg/apis/v1/nodepool_status.go

```diff
@@ -22,6 +22,9 @@ import (
 
 // NodePoolStatus defines the observed state of NodePool
 type NodePoolStatus struct {
+	// Nodes is the number of nodes maintained by ths NodePool
+	// +optional
+	Nodes int `json:"nodes,omitempty"`
```

Comment:
Why would this not be part of resources?

Reply:
Fair enough, I just moved it the resources fields